README.md

```diff
@@ -1,21 +1,134 @@
+## Getting Started
+---
+After cloning the repo and changing to it's directory, do the following.
 
+```bash
+npm i
+npm run build:dev
+```
+
+Deploy the bundled application to the player through the DWS APIs. See 
+[Enabling development tools on the Player ](#enable-player-development-tools) to enable the DWS, and [Deploying Code Remotely](#deploy-code) to deploy the code. 
+
+Alternatively, the bundled example, `dist/bundle`, `autorun.brs`, `index.html` can be copied to the root of the Micro SD card and insert into the player. Apply power to the player.
+
+## Preparing the Player for development
+---
+Your player needs to be setup with the console and the BrightScript debugger enabled to be used with the 3.5mm Serial Port and DWS. It's also usually a good idea to start with a player that is freshly factory reset.
+
+### Enabling development tools on the Player 
+### <a id="enable-player-development-tools"></a>
+
+`script debug on` will enable the BrightScript Debugger. Reboot the player and you will see logs on serial session. Press the SVC button and it will drop you to a BrightSign prompt.
+
+`registry write networking dwse on` will enable the DWS APIs to be accessible. Default password is the serial number of the player.
+
+`registry write html enable_web_inspector 1` will enable the Chromium Web Inspector.
+
+```
+script debug on
+registry write networking dwse on
+registry write html enable_web_inspector 1
+reboot
+```
+
+#### Verifying TX and RX functionality
+
+The BrightSign's log output when enabled, is sent over the 3.5mm Serial Port by default. 
+
+This example application can receive the player console either outputted through the 3.5mm or via telnet / ssh (see [Telnet and SSH](https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370673607/Telnet+and+SSH#Enabling-Telnet) for info on enabling either). The USB Serial Port will also log a message when it receives serial data -  for this application it will come from the 3.5mm console output or the Serial Port write command sending serial data over the 3.5mm port. 
+
+See [Deploying Code Remotely](#deploy-code) for info on how to deploy the code. 
+
+### Enabling Console
+
+This is not necessary, though receiving console output over a serial cable or Telnet / SSH connected to your PC gives insight into the application. Alternatively logs can be viewed in the DWS. You will need a serial cable and adapter for your computer, connected to the BrightSign's 3.5mm Serial Port.
+
+At the prompt, you can find the USB port by entering
+```bash
+ls /dev/tty*
+```
+
+For example, the USB port might be /dev/tty.usbserial in which case you can execute a command like,
+```
+screen /dev/tty.usbserial 115200 
+```
+
+Once connected, use the serial software of your choice (on linux and Mac, the "screen" command is quite popular and all you need).
```

Comment:
We should also add telnet/ssh instructions for Windows here.

Reply:
Thanks - I added a minor note. There's work being done to incorporate a more complete set of instructions to be published online.